app.js

```diff
@@ -1,109 +1,97 @@
-const express = require("express");
-const axios = require( 'axios' );
+import express from 'express';
+import {
+	isValidURL,
+	parseBoolOrInt,
+	parseScrapperResponse,
+	scrapLinks,
+} from './bin/utils.js';
+import {
+	parseValidatorResponse,
+	callHtmlValidator,
+	compileValidatorSummary,
+} from './bin/w3c-api.js';
+import { generateP2PostHtml, generateP2PostTags } from './bin/p2-post.js';
 
-const app = express();
-const log = console.log;
+const app = express(),
+	port = 3000;
 
-const { 
-    generateSummary,
-    tagsForP2Post,
-    generateHtmlPost,
-    scrapLinks
-} = require('./bin/utils');
+app.get( '/evaluate', async ( req, res ) => {
+	const url = req.query.url;
+	if ( ! url || ! isValidURL( url ) ) {
+		res.json( 'Please provide a valid URL' );
+	}
 
-app.listen(3000, () => {
-    log("Server running on port 3000");
-});
+	const format = req.query.format || 'p2html';
```

Comment:
I'm pretty sure there is only one place where we used `&output=json`, though I can't remember if it that place was the Monitor Tool or something else.

Could you please check and confirm if the `output` parameter was used there, and if that code has/will be updated to use `format` instead?

If the Monitor Tool is not the place where we use the `?output=` param, then I'd suggest adding backward compatibility by checking for `req.query.output` in case `req.query.format` is empty:
```
const format = req.query.format || req.query.output || 'p2html';
```

I agree that `format` is a better naming 👍

Reply:
It wasn't in the Team51 Monitor, so I just added your recommendation for backward compatibility. Thank you!